README: remove ../local/.. from auto-completion setup command (#181)
tldr-pages/tldr-python-client: README.md

```bash
# bash
tldr --print-completion bash | sudo tee "$BASH_COMPLETION_COMPAT_DIR"/tldr

> # zsh (recommend double checking where zsh/site-functions directory is located)

Author: ```suggestion
# zsh (it is recommended to check where zsh/site-functions directory is located)
```

# bash
tldr --print-completion bash | sudo tee "$BASH_COMPLETION_COMPAT_DIR"/tldr

> # zsh

Author: ```suggestion
```

# bash
tldr --print-completion bash | sudo tee "$BASH_COMPLETION_COMPAT_DIR"/tldr

> # zsh
> tldr --print-completion zsh | sudo tee /usr/share/zsh/site-functions/_tldr

Reviewer: So from the table concept, I think a better way may be something like:

```suggestion
# zsh (recommend double checking where zsh/site-functions directory is located)
# for macOS:
tldr --print-completion zsh | sudo tee /usr/local/share/zsh/site-functions/_tldr

# for Linux:
tldr --print-completion zsh | sudo tee /usr/share/zsh/site-functions/_tldr
```

Author: It will work however its not a fit all solution.

Reviewer: Well, make whatever change you think is right, I'm done offering suggestions.

> tldr --print-completion zsh | sudo tee /usr/share/zsh/site-functions/_tldr

Author: ```suggestion
# zsh (recommend double checking where zsh/site-functions directory is located)
## for macOS:
tldr --print-completion zsh | sudo tee /usr/local/share/zsh/site-functions/_tldr
## for Linux:
tldr --print-completion zsh | sudo tee /usr/share/zsh/site-functions/_tldr
```
